### PR TITLE
NXT-98 Rebrand

### PR DESCRIPTION
**Pull request by ediblecode**

### CODE REVIEW

File: web/src/test-utils.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
 import { HeadManagerContext } from "next/dist/shared/lib/head-manager-context";
 import { RouterContext } from "next/dist/shared/lib/router-context";
 import { NextRouter } from "next/router";
-import { ReactElement, useEffect, PropsWithChildren } from "react";
+import { ReactElement, useEffect, PropsWithChildren, ReactNode } from "react";
```
Comment (johndavey72) [inline web/src/test-utils.tsx:9]: Can't see ReactNode import in use anywhere in this file?

File: web/src/components/ToFromDateFilters/ToFromDateFilters.tsx [hunk 2]
```diff
@@ -41,120 +41,121 @@ export const ToFromDateFilters: FC<ToFromDateFilters> = ({
 );
 
 // Extra component to avoid the NDS filter group component passing invalid props (groupid/groupheading) into the div
-export const ToFromDateFiltersBody: FC<Except<ToFromDateFilters, "heading">> =
```
Comment (johndavey72) [inline web/src/components/ToFromDateFilters/ToFromDateFilters.tsx:44]: weird linebreak/cf formatting is showing up as a diff

**Review by johndavey72 — CHANGES_REQUESTED** [review]

**Comment by johndavey72** [comment]
Didn't notice any reference to colours changed as per AC2 of NXT-98?

File: web/src/components/ToFromDateFilters/ToFromDateFilters.tsx (hunk 2 above)
Comment (ediblecode) [reply web/src/components/ToFromDateFilters/ToFromDateFilters.tsx:44]: I _think_ that's a prettier thing TBH - nothing I've done. I agree it's annoying it's made GH think the whole thing's changed. It hasn't!

File: web/src/test-utils.tsx (hunk 1 above)
Comment (ediblecode) [reply web/src/test-utils.tsx:9]: Good spot. And `PropsWithChildren` actually. Will sort. You're like a human linter,

Comment (ediblecode) [reply web/src/test-utils.tsx]: Pushed a fix

**Comment by ediblecode** [comment]
> Didn't notice any reference to colours changed as per AC2 of NXT-98?

@johndavey72 that's because NXT doesn't actually use any NDS colours directly. It _does_ through GN. So a tester looking at this should see the new GN colours changes.

**Review by johndavey72 — DISMISSED** [review]

**Review by johndavey72 — APPROVED** [review]

**Pull request merged**